docs/development.rst

```diff
@@ -26,6 +26,10 @@ Quickstart
 - Install the :ref:`basic requirements <sec_installation_system_requirements>`.
 - Install the Python development requirements using ``pip install -r requirements/development.txt``.
 - Build the low level module by running ``make`` in the project root.
+
+.. note::
+   Mac OS X users should run ``make ext3`` instead.
```

Comment:
I think we should change the makefile here so that the default to build without lots of checks, and add a target ``allchecks`` which does the current default. This will make this advice unnecessary.

We can either do this here (if you're happy to do it) or create an issue to follow up on. Either way, we should remove this note and the other one at the end.

Reply:
yup, happy to look into how to do this - I've never made a makefile before but not afraid of having a poke around.

Reply:
I've just changed the order of the existing rules around -- so by default, `make` will run the `ext3` rule, and the `allchecks` rule (previously the default) is only run when explicitly specified by the user. Is this what you had in mind?

Reply:
Perfect, thanks.